common/services/user.js

```diff
@@ -1,4 +1,12 @@
 const axios = require('axios')
+const rax = require('retry-axios')
+
+const axiosInstance = axios.create()
+axiosInstance.defaults.raxConfig = {
+  instance: axiosInstance,
+  retries: 1,
```

Comment:
Should this be `retry`? The are the [options](https://github.com/JustinBeckwith/retry-axios/blob/97937ce75b8af78d60332ba8d4194df40645925d/src/index.ts#L15) I could see from their code, granted they don't seem to document them too well.

Reply:
Do we also want to make use of the [`statusCodesToRetry `](https://github.com/JustinBeckwith/retry-axios/blob/97937ce75b8af78d60332ba8d4194df40645925d/src/index.ts#L42) option to ensure it's only for 5xx responses? 

Looks like the default is `[[100, 199], [429, 429], [500, 599]]`, but maybe we're happy to retry on those too?

Reply:
Oops, yes, this should be `retry`, `retries` makes more sense though.

I think we should be fine with the default status codes, 429 is rate-limiting, 1xx appear to be things revolving around the request not being complete yet.